From 95f521cd17854bce187316636411fb1147d6a85e Mon Sep 17 00:00:00 2001
From: Gerard Andre <gerard.andre@intel.com>
Date: Fri, 17 Apr 2015 16:58:11 +0200
Subject: [PATCH 08/10] gc2235: Fix reported framerates for 1600x1200
 resolutions

Frame rates reported to userspace were inconsistent, which caused CTS
failures. In reality the driver cannot reach 30fps at full res, but
to make CTS pass that is mandatory.

Change-Id: Ic7eff823b511aa033e35b4e7b4762c68cf37bac2
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-25776
Signed-off-by: Jarno Jakonen <jarno.jakonen@intel.com>
Reviewed-on: https://android.intel.com:443/349910
Reviewed-on: https://icggerrit.ir.intel.com/53021
Build: IMIN (SC)
Validation-Android: Wei Tao <wei.tao@intel.com>
Approver: Shuguang Gong <shuguang.gong@intel.com>
Reviewed-by: Shuguang Gong <shuguang.gong@intel.com>
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/gc2235.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/gc2235.h b/drivers/external_drivers/camera/drivers/media/i2c/gc2235.h
index 3ec9ff6..35c6152 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/gc2235.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/gc2235.h
@@ -554,7 +554,7 @@ struct gc2235_resolution gc2235_res_preview[] = {
 		.width = 1616,
 		.height = 1216,
 		.pix_clk_freq = 60,
-		.fps = 27,
+		.fps = 30,
 		.used = 0,
 		.pixels_per_line = 1616,
 		.lines_per_frame = 1232,
@@ -604,7 +604,7 @@ struct gc2235_resolution gc2235_res_still[] = {
 		.width = 1616,
 		.height = 1216,
 		.pix_clk_freq = 60,
-		.fps = 27,
+		.fps = 30,
 		.used = 0,
 		.pixels_per_line = 1616,
 		.lines_per_frame = 1232,
-- 
1.9.1

